examples/C/color/rs-color.c

```diff
@@ -0,0 +1,127 @@
+// License: Apache 2.0. See LICENSE file in root directory.
+// Copyright(c) 2017 Intel Corporation. All Rights Reserved.
+
+/* Include the librealsense C header files */
+#include <librealsense2/rs.h>
+#include <librealsense2/h/rs_pipeline.h>
+#include <librealsense2/h/rs_frame.h>
+
+#include <stdlib.h>
+#include <stdint.h>
+#include <stdio.h>
+#include "../example.h"
+
+int main()
+{
+    rs2_error* e = 0;
+    /* Create a context object. This object owns the handles to all connected realsense devices. */
+    rs2_context* ctx = rs2_create_context(RS2_API_VERSION, &e);
```

Comment:
It might be helpful to state in comment for every rs2_create call how it is expected to be released. For instance, creates a context object, should be released with rs2_delete_context